sig-platform/charter.md

```diff
@@ -1 +1,43 @@
-To be created by SIG...
\ No newline at end of file
+# Charter
+
+This charter adheres to the conventions described in the [Open Data Hub Charter Guide](../sig-charter-guide.md) and uses the Roles and Organization Management outlined in [sig-goverance](../sig-governance.md).
+ 
+## Scope
+
+The Platform SIG maintains the core ODH operator, and establishes the standards and best practices for deployments, ecosystem integration, builds, monitoring, and secruity. 
+
+body of expertise necessary to ensure architectural consistency over time.
+
+## In scope
+
+- Standards for telemetry, integrations, testing, builds, DevOps, and Security
+- Deployments (embedded ODH, edge deployments, airgaped, embedded) scalability
+- ODH Operator
+- ODH Core Component Integration
+- Code, Binaries, Design and Services Covered
+- Opendatahub-io and opendatahub-io-contrib GitHub org
+- Monitoring, metrics
+- Platform Authentication/ Authorization
+- Service Mesh
+
+## Out of scope
+
+- Work covered under DevEx(UI/UX, Dashboard) and MLOps
+
+
+## Members:
+* Chairs: 
+  * Taneem Ibrahim
+  * Landon LaSmith
+  * Samu Veloso Lopez
+  * Gabe Goodheart
+  
+* Members: 
```

Comment:
This list doesn't match the list from above.

Reply:
@anishasthana List of people?

Reply:
Yeah. Huh, I could have sworn it looked a little different earlier. It matches now.